manifests/charts/ztunnel/templates/daemonset.yaml

```diff
@@ -40,14 +36,14 @@ spec:
           operator: Exists
       initContainers:
       - name: istio-init
-{{- if contains "/" .Values.ztunnel.image }}
-        image: "{{ .Values.ztunnel.image }}"
+{{- if contains "/" .Values.image }}
+        image: "{{ .Values.image }}"
 {{- else }}
         {{/*  TODO(https://github.com/istio/istio/issues/43243): use distroless, but we are depending on a variant of things not in the distroless image */}}
-        image: "{{ .Values.ztunnel.hub | default .Values.global.hub }}/{{ .Values.ztunnel.image | default "ztunnel" }}:{{ .Values.ztunnel.tag | default .Values.global.tag }}"
+        image: "{{ .Values.hub }}/{{ .Values.image | default "proxyv2" }}:{{ .Values.tag }}{{with (.Values.variant )}}-{{.}}{{end}}"
```

Comment:
```suggestion
        image: "{{ .Values.hub }}/{{ .Values.image | default "ztunnel" }}:{{ .Values.tag }}{{with (.Values.variant )}}-{{.}}{{end}}"
```

Reply:
The previous init used the ztunnel image, is there a reason to not?

Reply:
copy+paste error, thanks